packages/peregrine/lib/talons/OrderHistoryPage/useOrderHistoryPage.js

```diff
@@ -10,7 +10,7 @@ import { deriveErrorMessage } from '../../util/deriveErrorMessage';
 
 import DEFAULT_OPERATIONS from './orderHistoryPage.gql';
 
-const PAGE_SIZE = 10;
+const PAGE_SIZE = 2;
```

Comment:
Reason for smaller page size? Just to limit the amount of data?

Reply:
Just so it's easier to demo the behavior; would back this out before delivery.